changelog/new_add_internal_affairs_node_first_or_last_argument.md

```diff
@@ -0,0 +1 @@
+* [#12305](https://github.com/rubocop/rubocop/pull/12305): Add `InternalAffairs/NodeFirstOrLastArgument` cop. ([@sambostock][])
```

Comment:
The `InternalAffairs` department is internal API, it would be better to have a changelog like the one below:

```suggestion
* [#12305](https://github.com/rubocop/rubocop/pull/12305): Require RuboCop AST to 1.30 or higher. ([@sambostock][])
```

Reply:
I've updated it to

> Require `rubocop-ast` version 1.30 or greater